modules/xfeatures2d/src/harris_lapace_detector.cpp

```diff
@@ -4,6 +4,10 @@
 
 #include "precomp.hpp"
 
+#if defined __GNUC__
+#  pragma GCC diagnostic ignored "-Wunused-function"
```

Comment:
Maybe is it better to remove unused function?

Reply:
Fixed